docs/guide/rest-apis.md

```diff
@@ -190,7 +190,7 @@ In addition, we extend the REST convention with MongoDB-style operators:
 }
 ```
 
-### `DELETE /api/v1/product:docId`
+### `DELETE /api/v1/product/:id`
 
 **NOT supported. Instead PATCH the `trash` property to `true`.** Apostrophe never discards your work, so the `DELETE` verb is not applicable. However you may mark any piece as trash by `PATCH`ing that property. You may bring it back later by setting the property to `false`.
```

Comment:
Update: this is now supported. However if you do it in draft mode you should be aware that the published document will also be deleted, and that this is not the way to "discard a draft." You should also be aware that Apostrophe's general UI philosophy is not to delete things outright, except for drafts that have never been published, so PATCHing the `trash` property is often still the best policy.

Reply:
This is correct. In the interest of getting this out and keeping this focused on the locale and mode stuff I'd like to get to that in a future docs update. It's not likely scaring anyone off at the moment.

Reply:
I've created an internal ticket for this.